dev/tests/api-functional/testsuite/Magento/Sales/Service/V1/ShipmentGetTest.php

```diff
@@ -57,7 +58,14 @@ public function testShipmentGet()
         unset($data['tracks']);
         foreach ($data as $key => $value) {
             if (!empty($value)) {
-                $this->assertEquals($shipment->getData($key), $value, $key);
+                if($key === 'extension_attributes') {
+                    foreach ($value as $k => $val) {
+                        $methodName = 'get' . SimpleDataObjectConverter::snakeCaseToUpperCamelCase($k);
+                        $this->assertEquals($shipment->getExtensionAttributes()->$methodName(), $val);
+                    }
+                } else {
+                    $this->assertEquals($shipment->getData($key), $value, $key);
```

Comment:
the same Pull Request should be created for https://github.com/magento/magento2 github 

also, please provide cross-linkage between them